yarpcconfig/chooser_test.go

```diff
@@ -1060,6 +1062,70 @@ func TestChooserConfigurator(t *testing.T) {
 	}
 }
 
+func TestChooserConfiguratorWithoutPeerListUpdater(t *testing.T) {
+	tests := []struct {
+		desc    string
+		given   string
+		env     map[string]string
+		wantErr []string
+		test    func(*testing.T, yarpc.Config)
```

Comment:
Nit: `env` and `test` could be removed since the below test does not use it

Reply:
merged with previous table test instead.